pdr_backend/ppss/predictoor_ss.py

```diff
@@ -90,6 +90,12 @@ def s_until_epoch_end(self) -> int:
     def pred_submitter_mgr(self) -> str:
         return self.d["bot_only"]["pred_submitter_mgr"]
 
+    @property
+    def min_payout_slots(self) -> int:
+        if "min_payout_slots" not in self.d["bot_only"]:
```

Comment:
I would suggest: if "bot_only" not in self.d: return 0
return d["bot_only"].get("min_payout_slots", 0)

or a one-liner:
self.d.get('bot_only', {}).get('min_payout_slots', 0)